src/app/features/cases/components/CaseDetails/CaseDetails.tsx

```diff
@@ -4,15 +4,15 @@ import { useCase } from "app/state/rest"
 import { displayDate } from "app/features/shared/components/atoms/DateDisplay/DateDisplay"
 
 type Props = {
-  caseId: Components.Schemas.CaseEvent["id"]
+  caseId: MockComponents.Schemas.Case["id"]
```

Comment:
`Components.Schemas.Case["id"]` is the same type

Reply:
Fixed it